client/me/purchases/utils.js

```diff
@@ -92,7 +92,6 @@ function getEditCardDetailsPath( siteSlug, purchase ) {
 
 export {
 	getPurchase,
-	getSelectedSite,
```

Comment:
Maybe also replace the single usage of `getSelectedSite()` inside this file and remove `getSelectedSite()` completely?

Reply:
On the roadmap, but I'd rather leave the function alone so I don't introduce a conflict with #24799. I'll get it after.